pyproject.toml

```diff
@@ -0,0 +1,48 @@
+[tool.poetry]
+name = "vc-authn-oidc"
+version = "0.2.2"
+description = "Verifiable Credential Identity Provider for OpenID Connect."
+authors = ["TODO"]
+license = "Apache-2.0"
+readme = "README.md"
+classifiers = [
+            "Programming Language :: Python :: 3",
+            "License :: OSI Approved :: Apache Software License",
+	        "Operating System :: OS Independent",
+]
+repository = "https://github.com/bcgov/vc-authn-oidc"
+
+[tool.poetry.dependencies]
+python = ">=3.11,<4.0"
+fastapi = "0.111.0"
+jinja2 = "3.1.4"
+oic = "1.7.0"
+pymongo = "4.7.3"
+pyop = "3.4.1"
+python-multipart = "0.0.9" # required by fastapi to serve/upload files
+qrcode = { version = "7.4.2", extras = ["pil"]}
+structlog = "24.2.0"
+uvicorn = { version = "0.30.1", extras = ["standard"] }
+python-socketio = "5.11.3" # required to run websockets
+canonicaljson = "2.0.0" # used to provide unique consistent user identifiers"
+pydantic-settings = "2.3.3"
+
+[tool.poetry.group.dev.dependencies]
+black = "^24.4.2"
+mock = "^5.1.0"
+mongomock = "4.1.2"
+pytest-asyncio = "0.23.7"
+pytest-cov = "5.0.0"
+pytest = "8.2.2"
+requests-mock = "1.12.1"
+setuptools = "^75.1.0"
```

Comment:
Please use semantic versioning allowing minor version updates, rather than pinning dependencies.
If possible, update dependencies as well to the latest version (unless major refactoring work is required).